lib/ex_doc/language.ex

```diff
@@ -131,6 +131,15 @@ defmodule ExDoc.Language do
               opts: keyword()
             }
 
-  def get(:elixir), do: ExDoc.Language.Elixir
-  def get(:erlang), do: ExDoc.Language.Erlang
+  def get(:elixir, _module), do: {:ok, ExDoc.Language.Elixir}
+  def get(:erlang, _module), do: {:ok, ExDoc.Language.Erlang}
+
+  def get(language, module) do
+    IO.puts(
+      :stderr,
+      "warning: skipping module in unsupported language #{inspect(language)} : #{module}"
+    )
```

Comment:
^ I picked `IO.puts/2` since I saw it elsewhere on the codebase (and I didn't see any `Logger` calls.)

Reply:
You can use `IO.warn/1`.

Reply:
Right! I discovered it in the mean time :smile: but it prints the stacktrace along with the message. Is that desirable in this particular case?

Reply:
Switched to `IO.warn/2` with an empty stacktrace in 3fd39a20, as suggested by @josevalim .